lib/Dialect/TTKernel/IR/TTKernelOps.cpp

```diff
@@ -91,6 +91,78 @@ ::mlir::LogicalResult CBWaitFrontOp::verify() {
   return success();
 }
 
+::mlir::LogicalResult TilizeInitOp::verify() {
+  if (not insideDispatchOpRegion(getOperation())) {
+    return emitOpError("TilizeInitOp must be inside of a DispatchOp region");
+  }
+  if (getCbIn().getType().getPort() == getCbOut().getType().getPort()) {
+    return emitOpError("cbIn and cbOut must be different");
```

Comment:
I'd be more clear when emitting errors, for example here I'd say:
`Input circular buffer port and output circular buffer port must be different`

Same for the remainder of errors in the file.